manon/components/checkbox.scss

```diff
@@ -47,41 +31,37 @@
       }
     }
   }
+}
 
-  &.required {
-    flex-direction: column;
-    align-items: flex-start;
+@mixin checkbox {
+  .checkbox {
+    padding: 0;
+    margin: 0 0 0.5rem;
```

Comment:
We want the theme builders to be able to set any visual styling. We also want to be as least opinionated with base styling as we can be. Add it as a variable instead ideally without a base value. (default can be set to null)